packages/create-react-native-library/templates/common/$package.json

```diff
@@ -100,7 +112,12 @@
   },
   "peerDependencies": {
     "react": "*",
+<% if (project.nitro) { -%>
+    "react-native": "*",
+    "react-native-nitro-modules": "*"
```

Comment:
Will the library stay compatible with multiple versions of `react-native-nitro-modules`?

Reply:
I guess this would be a question for @mrousavy. Are you folks planning to introduce breaking changes in the future? Will it always use zero based versioning? Should we use some version constraints here?

Reply:
But, I think we shouldn't worry about this too much tho, people don't care about the peer dependency warnings at all